tests/p256_ecdsa.cc

```diff
@@ -96,6 +96,7 @@ testImplementationHacl()
     Hacl_P256_ecdsa_sign_p256_without_hash(result, 32, digest, prKey, nonce);
   bool s0 = compare_and_print(32, result, siggen_vectors_low5);
   bool s1 = compare_and_print(32, result + 32, siggen_vectors_low6);
+  free(result);
   return s0 && s1 && flag;
```

Comment:
Should this also be `KRML_HOST_FREE()` or would that only be cosmetic?

Reply:
It's not different https://github.com/cryspen/hacl-packages/blob/d15cdb379c14b2ef816ca7c603972ff277cae008/karamel/include/krml/internal/target.h#L46